botfront/imports/api/graphql/conversations/resolvers/conversationsResolver.js

```diff
@@ -28,6 +32,35 @@ export default {
             const response = await deleteConversation(args.id);
             return { success: response.ok === 1 };
         },
+        importConversations: async (_, args, __) => {
+            const { conversations, projectId, environment } = args;
+            const latestAddition = await Conversations.findOne({ env: environment, projectId })
+                .select('tracker.latest_event_time')
+                .sort('-tracker.latest_event_time')
+                .lean()
+                .exec();
+            const latestTimestamp = latestAddition
+                ? latestAddition.tracker.latest_event_time : 0;
+            const results = await Promise.all(conversations
+                .filter(c => c.tracker.latest_event_time >= latestTimestamp)
+                .map(c => upsertTrackerStore({
+                    senderId: c.tracker.sender_id || c._id,
+                    projectId,
+                    env: environment,
+                    tracker: c.tracker,
+                    overwriteEvents: true,
```

Comment:
I'm not sure to understand why we might want to overwrite the events, can you shine some light on this ?